components/03-sections/02-navigation/04-site-footer/site-footer.readme.md

```diff
@@ -8,4 +8,9 @@ This section utilizes existing bootstrap classes for layout and simple FontAweso
 
 ## Type of information
 
-The site footer is the last chance for presenting site contact information (College address, phone, email, etc).  It should be standard to include this immediately before the global UTSA footer.  An exception for not displaying this footer can be made for custom landing pages or one pagers.
\ No newline at end of file
+The site footer is the last chance for presenting site contact information (College address, phone, email, etc).  It should be standard to include this immediately before the global UTSA footer.  An exception for not displaying this footer can be made for custom landing pages or one pagers.
+
+## Media Requirements
+Site Footer Image: The image in the site footer. Our standard image size is 1600px (width) by 500px (height), 72 ppi (pixels per inch) jpeg or png.
```

Comment:
for the footer I was using this as reference: https://walledev.it.utsa.edu/entity/open.act?id=420d837d81736a1b55fcab75ae19a99d&type=folder

and for the 460x210 dimensions I was using the last slide of this: https://xd.adobe.com/view/f9527e25-3f4f-4ad4-b2c3-b7e14504c490-ac01/screen/4d8a19f0-3b57-4808-bfc0-d8b1983be2ea

Reply:
I was told by uriel that news/events don't have placeholder sizes because the images get pulled in with the article

Reply:
@bennie-aranda Let's remove the events and news to avoid confusion or specify that it's only "Displays at" size.

The footer information should be listed on this page:
https://utsa-asc.github.io/college-dls/566-dls-image-requirements-need-to-be-added-to-all-components-that-have-an-image/components/detail/enrollment-call-to-action--default.html

It's being displayed on this page, which was confusing to me:
https://utsa-asc.github.io/college-dls/566-dls-image-requirements-need-to-be-added-to-all-components-that-have-an-image/components/detail/site-footer.html

Reply:
I am a bit confused, do I put the footer "media requirements" on the enrollment CTA tab because atm the footer "media requirements" are taking place in the "site footer" tab.

Reply:
did you want me to add the cta requirements of 600x400?

Reply:
Yes, the footer shouldn't have the media requirements, but it should be on the enrollment CTA:
https://utsa-asc.github.io/college-dls/566-dls-image-requirements-need-to-be-added-to-all-components-that-have-an-image/components/detail/enrollment-call-to-action--default.html

It looks like the CTA info has the media requirements:
https://utsa-asc.github.io/college-dls/566-dls-image-requirements-need-to-be-added-to-all-components-that-have-an-image/components/detail/call-to-action-media--default.html

It looks like it isn't in a list like the others, though:
https://utsa-asc.github.io/college-dls/566-dls-image-requirements-need-to-be-added-to-all-components-that-have-an-image/components/detail/action-card--default.html